Add import plugin tutorial

vnen opened the pull request:
This is based on the import plugin demo.

vnen commented on development/plugins/import_plugins.rst:
```diff
@@ -0,0 +1,410 @@
+    ...
+
+Now that the enum is defined, let's keep looking at the methods of an import
+plugin::
```
I'm gonna properly review everything tomorrow, but I just didn't get this one. Where is the colon and line break missing?

mhilbrunner replied:
Current:
```
Now that the enum is defined, let's keep looking at the methods of an import
plugin::

    func get_preset_count():
```

Should probably be:
```
Now that the enum is defined, let's keep looking at the methods of an import
plugin:

::

    func get_preset_count():
```

akien-mga replied:
The style used is valid too (but I tend not to use it as it makes it harder to spot actual code blocks IMO).

mhilbrunner replied:
Ah, nervermind then, sorry :)

vnen commented:
Updated, changed a bit more than the suggestions for some stuff still sounded unclear to me.

Also changed all the code blocks to use the standalone `::` line, since this seems preferable.
